komga-webui/src/components/ThumbnailCard.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div class="col-6 col-lg-3 pa-1">
```

Comment:
it would be best to remove this from the component, it should be the responsability of the parent component to decide on the sizing